src/common/Pagination/index.js

```diff
@@ -17,25 +17,25 @@ export const Pagination = () => {
   return (
     <Wrapper>
       <Button onClick={setFirstPage} disabled={isFirstPage}>
-        <PrevArrowIcon isDisabled={isFirstPage} />
```

Comment:
React throws a warning that: it does not recognize the "isDisabled" property in the DOM element. Despite this, it works fine. Shouldn't all references be replaced with "isDisabled" or given another neutral name?

Reply:
You commented an outdated piece of code. I have already resolved this in a recent commit.

Reply:
One more error is shown (Warning: Maximum update depth exceeded. This can happen when a component calls setState inside useEffect, but useEffect either doesn't have a dependency array, or one of the dependencies changes on every render), which is available in version, but pagination works fine.

Reply:
It also has been resolved in a recent commit. You somehow looked up on old code before my commits from today.

Reply:
> React throws a warning that: it does not recognize the "isDisabled" property in the DOM element. Despite this, it works fine. Shouldn't all references be replaced with "isDisabled" or given another neutral name?

Now after my changes, a warning is shown that:
react-dom.development.js:86 Warning: Received `true` for a non-boolean attribute `isdisabled`.

And that I should make it as a "string" value instead of boolean, but if I change that value to string, arrow icons doesn't work properly and they stay "disabled" all the time and stay grey.
isdisabled refers to an attribute that triggers some styles when also the button itself is disabled, thats why there is a different name to that

Reply:
Apart from that, good job.